cmake/nsis/build_mingw_drumstick.sh

```diff
@@ -0,0 +1,54 @@
+drumstick_ver=$1
```

Comment:
This can stay as-is, but this should probably be ported to https://github.com/LMMS/lmms/blob/master/cmake/msys/msys_helper.sh like @DragonEagle did in #3369 or alternately added directly to the mingw mirros that Toby maintains for us.  Toby generally does this on-request.

I think first though we can pull Qt4 support, which is an open task, so we wouldn't hold up the merge for this script alone.